### ISSUE TITLE
can't open config file: permission denied

### ISSUE THREAD

**Issue by olivluca** [Rendevior/mpv-android-argparse#5]
I'm trying your script but

```
08-23 18:14:10.985  7535 31999 V mpv     : [cplayer:v] Reading config file /storage/emulated/0/mpv/mpv.tmp
08-23 18:14:10.985  7535 31999 V mpv     : [cplayer:v] Can't open config file: Permission denied

```
mpv has access to /storage/emulated/0 and it can play files from there (e.g. I tried playing successfully the recording_20220404_090617.mp4 file there)

```
J606F:/ $ ls -l /storage/emulated/0/mpv
total 1508
-rw-rw---- 1 root everybody     597 2022-08-23 18:14 mpv.tmp
-rw-rw---- 1 root everybody 1539505 2022-08-23 18:18 recording_20220404_090617.mp4


```

**Comment by olivluca** [comment]
I found out that mpv only has permission to access multimedia files (under the app permission settings I can only give it no permission at all or just multimedia files).
I added .mp4 to the name of the file and now it works.
BTW, since I'm using this script to pass the subtitles from lobster I had do remove the sed line that removes the \\ character.

**Comment by TheGlockMisc** [comment]
> I found out that mpv only has permission to access multimedia files (under the app permission settings I can only give it no permission at all or just multimedia files). I added .mp4 to the name of the file and now it works. BTW, since I'm using this script to pass the subtitles from lobster I had do remove the sed line that removes the \ character.

I never encountered that permission error once, Renaming file with .mp4 is neat just to bypass the restriction of multimedia files only. Btw, Does lobster pass subtitle urls with backslashes?

**Comment by olivluca** [comment]
Yes, to escape the ":" between http/https and the rest of the url (mpv uses ":" as a separator between urls :sob: so it would interpret https://something as 2 urls, https and something, none of them working)

**Comment by olivluca** [comment]
Oh, and regarding the permission, I just installed mpv on my phone (android 10) and the permission is "access to storage, allow/deny", while in android 11 is "access to storage, allow only access to multimedia files/deny" (rughly translated from my localized version).

**Comment by TheGlockMisc** [comment]
> Yes, to escape the ":" between http/https and the rest of the url (mpv uses ":" as a separator between urls 😭 so it would interpret https://something as 2 urls, https and //something, none of them working)

Ooof. I haven't thought of colon delimiter😅. let me remove that line strips backslash.

**Comment by olivluca** [comment]
(and I suppose the same problem would apply to the log file)

**Comment by Rendevior** [comment]
I guess, this was solved now.

**Comment by olivluca** [comment]
I didn't investigate further, but probably this affects also the log file, not only the configuration file (fixed in 14dd27910fc8593792ef4ab0b2ebb381e2e6fb00)
